### PR TITLE
Improved PR template

### PR DESCRIPTION
**Pull request by saarthakchats**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`
- Preview Link
-

## Description / Changes

- Commented the parts which need to be read, but not shown
- Added more understandable explainations
- Added reference link for squashing
- Changed main branch to 'master'
- Added space for screenshots
- Hint for Preview Link

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,19 +1,34 @@
-Please read and understand everything below
-**Do not delete any text other than where you are instructed.**
+<!-- Please read and understand everything below
+**Do not delete any text other than where you are instructed.** -->
```
Comment (AakashMallik) [inline .github/PULL_REQUEST_TEMPLATE.md]: other than where you are instructed.  => other **than the ones you are intsructed to**.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
 
-**Students: If one of them is applicable to you. Please check it.**
+<!-- **Students: If one of them is applicable to you. Please check it.** -->
 
-Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.
+<!-- Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.-->
 
-- [ ] Included a Preview link and screenshot showning after and before the changes.
+- [ ] Included a Preview link and screenshot showing after and before the changes.
 - [ ] Included a description of change below.
```
Comment (AakashMallik) [inline .github/PULL_REQUEST_TEMPLATE.md]: of **the** change

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
+## Screenshots if any:
+### Before:
+
+### After:
+
+
+
+- - - - - - - - - - - -
+<!-- [preview link url]: https://<github_username>.github.io/<name_of_repository> -->
+<!-- [squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git] -->
```
Comment (fragm3) [inline .github/PULL_REQUEST_TEMPLATE.md]: Use this, its better: https://davidwalsh.name/squash-commits-git

**Comment by saarthakchats** [comment]
@fragm3 Done,
Please merge

**Comment by saarthakchats** [comment]
Is this good to merge now?
@nikit19 @fragm3

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (AakashMallik) [reply .github/PULL_REQUEST_TEMPLATE.md]: @saarthakchats Please be careful while editing. That was not the suggested change. It is grammatically incorrect.

**Pull request merged**